src/packagers/HTMLPackager.js

```diff
@@ -44,6 +60,27 @@ class HTMLPackager extends Packager {
       });
     }
   }
+
+  insertJSBundle(jsBundle, tree) {
+    let head = find(tree, 'head');
+    if (!head) {
+      let html = find(tree, 'html');
+      head = {tag: 'head'};
+      html.content.unshift(head);
+    }
+
+    if (!head.content) {
+      head.content = [];
+    }
```

Comment:
maybe the above code can be moved to another method/function and shared with the css version?

Reply:
Probably a good idea, i'll look into that